src/components/FeatureViews/Registration/RegistrationCourseEnrollments.js

```diff
@@ -37,12 +48,28 @@ const TableToolbar = (
 );
 
 const RegistrationCourseEnrollments = ({courseID}) => {
+    const dispatch = useDispatch();
+    const api = useMemo(
+        () => ({
+            ...bindActionCreators(registrationActions, dispatch),
+        }),
+        [dispatch]
+    );
+
     const [expanded, setExpanded] = useState({});
     const courses = useSelector(({"Course": {NewCourseList}}) => NewCourseList);
     const parents = useSelector(({"Users": {ParentList}}) => ParentList);
     const students = useSelector(({"Users": {StudentList}}) => StudentList);
     const enrollments = useSelector(({Enrollments}) => Enrollments);
 
+    const [studentMenuAnchorEl, setStudentMenuAnchorEl] = useState(null);
+    const [unenroll, setUnenroll] = useState({
+        open: false,
+        student: null,
+        course: null,
```

Comment:
Can you remove these fields from state and just pull it from the enrollment when needed?